tests/c.test

```diff
@@ -377,6 +377,8 @@
 09615  i1564.cfg                            c/i1564.c
 
 09616  enum_comma_ifdef.cfg                 c/enum_comma_ifdef.c
+09617  09617.cfg                            c/issue_2360.c
+09618  09618.cfg                            c/issue_2360.c
 
```

Comment:
Please use the current way for the file name:
Change at tests/c.text:
09617.cfg to Issue_2360-a.cfg
09618.cfg to Issue_2360-b.cfg
c/issue_2360.c to c/Issue_2360.c